model/reader.go

```diff
@@ -765,6 +765,25 @@ func (r *PdfReader) GetNamedDestinations() (core.PdfObject, error) {
 	return obj, nil
 }
 
+// GetPageLabels returns the PageLabels entry in the PDF catalog.
+// See section 12.4.2 "Page Labels" (p. 382 PDF32000_2008).
+func (r *PdfReader) GetPageLabels() (core.PdfObject, error) {
+	obj := core.ResolveReference(r.catalog.Get("PageLabels"))
+	if obj == nil {
+		return nil, nil
+	}
+
+	// Resolve references.
+	if !r.isLazy {
```

Comment:
Would it be sufficient to call core.GetDict() ?  This object is a dictionary right?

Reply:
The method is implemented in the same way as the other similar methods in the reader: `GetOCProperties`, `GetNamedDestinations`, which traverse the output object and resolve children references in non-lazy mode. The elements inside the `Nums` array of the `PageLabels` dictionary can be references.